lib/modules/manager/gradle/extract.ts

```diff
@@ -101,6 +103,7 @@ export async function extractAllPackageFiles(
     // istanbul ignore else
     if (key) {
       let pkgFile = packageFilesByName[key];
+      // istanbul ignore if: won't happen if "apply from" processes only initially known files
```

Comment:
but files can be missing from initial file match when not included in match regex or ignored.

Reply:
Not sure if I understand you correctly. In the rewrite, `apply from` allows only inclusion of `packageFiles: string[]` initially passed to `extractAllPackageFiles()`. It shouldn't include or non-existing ignored files, as renovate already filtered them beforehand. Am I missing something here?

Reply:
ok